api/openapi-spec/v1.0.yaml

```
@@ -493,6 +493,40 @@ paths:
         default:
           $ref: '#/components/responses/error'
       x-ms-docs-operation-type: operation
+  '/groups/{group-id}/members':
+    get:
+      tags:
+        - group
+      summary: Get a list of a groups members
+      operationId: GetMembers
```

Is there a graph api link that calls this GetMembers? I prefer ListMembers, but maybe that's just me
```suggestion
      operationId: ListMembers
```